funnel/views/event.py

```diff
@@ -209,4 +209,6 @@ def event_checkin(profile, space, event, participant):
     a.checked_in = checked_in
     db.session.add(a)
     db.session.commit()
+    if request.method == 'POST':
```

Comment:
Use `if request.xhr` here. Using jQuery's `$.ajax` sets the appropriate HTTP header and Flask will automatically set `request.xhr` to `True`.